cer-graphql/package.json

```diff
@@ -5,6 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "jest",
+    "test-dev": "export awsProfile='dev' && npm run test",
+    "test-nonprod": "export awsProfile='test' && npm run test",
```

Comment:
You will also need to modify the Jenkinsfile stage 'Run cer-graphql tests' to use a similar command so that it gets the correct stage and aws profile

Reply:
You can do this to get the correct stage from the branch name: 
def stage = (
      env.BRANCH_NAME == 'prod' ? 'prod' : 
      env.BRANCH_NAME == 'nonprod' ? 'test' : 
      'dev'
  )

Reply:
Also Jenkins should only run the tests if it's not on prod branch

Reply:
Nice little ternary there @rosemcc 👍 😍